src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final String[] colors = {"Red", "Green", "Blue", "White", "Black", "Orange"};
```

Comment:
It's better to put the colors in a separate enum. There will be no extension in this task, but in general, if you still need to use colors somewhere, you will either have to duplicate our array or transfer it to enam later, rewriting the old code, which is not very good. It is better to immediately learn to do more scalable.